File: go-sdk/pkg/models/operations/getadeployment.go
package operations

type GetADeploymentPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type GetADeploymentRequest struct {
	PathParams GetADeploymentPathParams
}

type GetADeploymentResponse struct {
	ContentType                            string
	StatusCode                             int
	GetADeployment200ApplicationJSONObject map[string]map[string]interface{}
}

File: ts-sdk/src/sdk/models/operations/getadeployment.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetADeploymentPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}


export class GetADeploymentRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetADeploymentPathParams;
}


export class GetADeploymentResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getADeployment200ApplicationJSONObject?: Record<string, Record<string, any>>;
}
